welcome.php

```
@@ -1,6 +1,7 @@
 <?php
 // Initialize the session
-session_start();
+// Added the @ to tell PHP to ignore session_start if called multiple times
+@session_start();
```

I don't understand this part. I thought that the @ symbol was used to suppress errors?

---

Indeed it is. If you have multiple session_start lines in your code (for example, because multiple people work on the same file and may add the command without checking if the session is already initiated), the @ suppresses all the session_start commands beyond the first one.  

Reference: https://youtu.be/vu9-IymiOAQ?t=818